_ptbr/letter-three.md

```diff
@@ -25,12 +26,12 @@ title: Terceira carta
   </div>
 
   <div class="LetterThree-block LetterThree-block--two">
-    Ainda não temos <img 
+    Aqui os <img 
```

Comment:
```suggestion
    Aqui, os <img 
```
?

Reply:
Hm... I think such a comma adds a pause in this sentence that doesn't reflect the tone from the english text.